examples/hetzner-linux/main.tf

```diff
@@ -0,0 +1,117 @@
+terraform {
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = "0.4.1"
+    }
+    hcloud = {
+      source  = "hetznercloud/hcloud"
+      version = "1.33.2"
+    }
+  }
+}
+
+provider "hcloud" {
+  token = var.hcloud_token
+}
+
+provider "coder" {
+}
+
+variable "hcloud_token" {
+  description = <<EOF
+Coder requires a Hetzner Cloud token to provision workspaces.
+EOF
+  sensitive   = true
```

Comment:
Suggestion: We could try to validate the length/format here to ensure it's the right kind of token.

Reply:
I added a check for this. Basically all hetzner cloud token are 64 chars long, so I simply check for this :)